### DISCUSSION TITLE
Distinction between hard and soft skills

### DISCUSSION THREAD

**Discussion by LidiaKovac** [AnasAito/SkillNER#43] [Q&A] [answered]
Hello! Here with another query about this wonderful module! I have successfully translated and analyzed the text (thanks for the help!), but it is unclear to me how to tell hard skills from soft skills. I understand this module is built upon a skills database, do I maybe need to reference that as well? From the examples and docs as well as my results I cannot find a pattern. 
Thanks in advance!

**Comment by AnasAito** [comment, marked as answer]
Hello @LidiaKovac  Yes in fact we're using a knowledge base of skills to link extracted entities from text to unique skills and avoid duplication , for the reference we're using EMSI skill database where they give metadata for their skills (description and skill type : hard skill , soft skill or certification) 
you can explore the knowledge base here : https://skills.emsidata.com/
